qa/setup_packages.py

```diff
@@ -324,7 +324,8 @@ def get_pyvers_name(self, url, cuda_version):
                         { "100" : [
                               PckgVer("1.15.2",  python_max_ver="3.7"),
                               PckgVer("2.1.1",  python_max_ver="3.7"),
-                              "2.2.0"]
+                              "2.2.0",
+                              "2.3.0"]
```

Comment:
With regards to this, what do you think about dropping tests for 2.1.1? And possibly moving from 1.15.2 to 1.15.3? Almost no difference there, but it is the latest version.

Reply:
I'm for both ideas.

Reply:
Ok, done.